### PR TITLE
Sury rulez.

### PR DESCRIPTION
**Pull request by athoune**
See #40

### CODE REVIEW

File: Dockerfile.7.x-cli [hunk 1]
```diff
@@ -9,6 +9,7 @@ ARG PHP_VERSION
 LABEL php.version=${PHP_VERSION}
 RUN set -eux \
     &&  export http_proxy=${HTTP_PROXY} \
+    &&  if [[ $PHP_MINOR_VERSION -gt 1 ]]; then echo "PHP_VERSION > 7.1" && printf "Package: *\nPin: origin packages.sury.org\nPin-Priority: 100\n" > /etc/apt/preferences.d/php ; fi \
```
Comment (zerodeux) [inline Dockerfile.7.x-cli:12]: Condition/test not necessary : this recipe uses Sury's repo thus always need the APT pinning (for every version provided by Sury).

File: Dockerfile.7.x-cli [hunk 2]
```diff
@@ -24,9 +25,11 @@ RUN set -eux \
     &&  touch /etc/msmtprc \
     &&  chmod 640 /etc/msmtprc \
     &&  curl https://packages.sury.org/php/apt.gpg -o /etc/apt/trusted.gpg.d/php-sury.gpg \
-    &&  echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | tee -a /etc/apt/sources.list.d/php-sury.list \
+    &&  echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php-sury.list \
     &&  apt-get update \
+    &&  cat /etc/apt/preferences.d/php \
```
Comment (zerodeux) [inline Dockerfile.7.x-cli:30]: Why ? For debug purposes ?

File: Dockerfile.7.x-cli [hunk 3]
```diff
@@ -24,9 +25,11 @@ RUN set -eux \
...
     &&  apt-get install -y --no-install-recommends \
+                  php7.${PHP_MINOR_VERSION}-common \
```
Comment (zerodeux) [inline Dockerfile.7.x-cli:32]: Not necessary, SAPIs like php7.x-cli always depend on php7.x-common, no need to explictly install it.

**Review by zerodeux — CHANGES_REQUESTED** [review]
I left 3 minor comments.

File: Dockerfile.7.x-cli (hunk 1 above)
Comment (athoune) [reply Dockerfile.7.x-cli:12]: It's a debug for the pinning bug.

File: Dockerfile.7.x-cli (hunk 2 above)
Comment (athoune) [reply Dockerfile.7.x-cli:30]: yes, debug.

File: Dockerfile.7.x-cli (hunk 3 above)
Comment (athoune) [reply Dockerfile.7.x-cli:32]: it's a loosy try to fix pinning bug.

**Comment by athoune** [comment]
It's broken, see #40

**Pull request closed without merging**